packages/uiweb/src/lib/helpers/udResolver.ts

```diff
@@ -3,23 +3,27 @@ import Resolution from '@unstoppabledomains/resolution';
 import { ethers } from 'ethers';
 import { allowedNetworks, InfuraAPIKey, NETWORK_DETAILS } from '../config';
 
-export const getUdResolver = (env:Env): Resolution => {
-  const l1ChainId = allowedNetworks[env].includes(1) ? 1 : 5;
-  const l2ChainId = allowedNetworks[env].includes(137) ? 137 : 80002;
-  // ToDo: Enable for sepolia chainId once UD supports it
-  // const l1ChainId = appConfig.allowedNetworks.includes(1) ? 1 : 11155111;
-  return Resolution.fromEthersProvider({
-    uns: {
-      locations: {
-        Layer1: {
-          network: "mainnet", // add config for sepolia once it's supported by UD
-          provider: new ethers.providers.InfuraProvider(l1ChainId, InfuraAPIKey),
-        },
-        Layer2: {
-          network: NETWORK_DETAILS[l2ChainId].network,
-          provider: new ethers.providers.InfuraProvider(l2ChainId, InfuraAPIKey),
+export const getUdResolver = (env: Env): Resolution => {
+  try {
+    const l1ChainId = allowedNetworks[env].includes(1) ? 1 : 5;
+    const l2ChainId = allowedNetworks[env].includes(137) ? 137 : 80002;
```

Comment:
General Question?:

What's 1,5,137, 80002?

For now is it possible to have string values instead of these?

If no can we atleast store them into const which will help us giving them proper names and reuse across files.

Reply:
keeping it numbers because ud doesnot have follow the constants defined, these are specifically used for ud not reused anywhere @rohitmalhotra1420

Reply:
Everywhere else in the sdk we use a constant for determining chain-id but for ud its kind of an exception, as it doesnot support the test l1 and l1 chains

Reply:
@mishramonalisha76 and this one as well